src/it/java/servlets/admin/config/EnableCheatsIT.java

```diff
@@ -1,271 +1,271 @@
-package servlets.admin.config;
-
-import static org.junit.Assert.*;
-
-import org.apache.log4j.Logger;
-import org.junit.Before;
-import org.junit.BeforeClass;
-import org.junit.Test;
-import org.springframework.mock.web.MockHttpServletRequest;
-import org.springframework.mock.web.MockHttpServletResponse;
-import org.springframework.mock.web.MockServletConfig;
-
-import testUtils.TestProperties;
-import utils.CheatSheetStatus;
-import utils.InstallationException;
-
-public class EnableCheatsTest
-{
-	private static org.apache.log4j.Logger log = Logger.getLogger(EnableCheats.class);

```

Comment:
EnableCheats.class should also be renamed to EnableCheatsIT (mistake in the Test class too)